src/rust/ide/src/double_representation/module.rs

```diff
@@ -0,0 +1,143 @@
+//! Code for module-level double representation processing.
+
+use crate::prelude::*;
+
+use crate::double_representation::definition;
+use crate::double_representation::definition::DefinitionProvider;
+
+use ast::crumbs::ChildAst;
+use ast::known;
+use enso_protocol::language_server;
+
+
+
+// ==============
+// === Errors ===
+// ==============
+
+#[allow(missing_docs)]
+#[derive(Fail,Clone,Debug)]
+#[fail(display="Cannot find method by pointer {:?}.",_0)]
```

Comment:
If I understand correctly, we can "query something by ..." or "lookup something by ...", but here the "by" should rather not be used. Instead it should be "of" or "with". Please confirm with @iamrecursion

Reply:
changed to `with`